docs/roadmap.md

```diff
@@ -11,37 +11,49 @@ Here's how we intend to roll out Project Reunion:
 <table>
   <tbody>
     <tr>
-      <th>//BUILD 2020</th>
-      <th>End of 2020</th>
-      <th>2021++</th>
+      <th>Reunion 0.5 (March 2021)</th>
+      <th>Reunion 0.8 (May 2021)</th>
+      <th>Reunion 1.0 (October 2021)</th>
+      <th>Past October Release</th>
     </tr>
     <tr>
-      <td>
+    <td>
+      <ul>
+            <li>1st supported public release</li>
+            <li>UWP features available to unpackaged Win32</li>
+            <li>Enabling developers to easily have their apps updates and repair</li>
```

Comment:
Reading this makes me think we're providing some packaging/updating solution for apps to update themselves, but I'm not sure we're doing that?

Maybe this is more an internal detail of "We can update the SDK without apps making changes themselves?" Something needs to clarify that this is only scoped to the Reunion SDK updating without app dev work